src/index.js

```diff
@@ -46,47 +44,58 @@ module.exports = {
     'id-length': ['error', {
       exceptions: ['_', 'e', 'i']
     }],
+    'id-match': ['error', '^_$|^_[a-z]+[a-zA-Z]*$|^[A-Z][a-z]+[a-zA-Z]*$|^[A-Z]+_[A-Z]+$|^[a-z]+[A-Za-z]*$'],
```

Comment:
This regular expression sucks, it's basically just a placeholder. We need to see what kind of variable naming convention we will use and what exactly it will allow.